ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+**Do you want to request a *feature* or report a *bug*?**
+
+**What is the current behavior?**
+
+**If the current behavior is a bug, please provide the steps to reproduce.**
+<!-- A great way to do this is to provide your configuration via a GitHub gist. -->
+
+**What is the expected behavior?**
+
+**If this is a feature request, what is motivation or use case for changing the behavior?**
+
+**Local Environment Information**
+Paste the results of `npx envinfo` here
```

Comment:
I don't think it's always required... Maybe say something around, "if this issue has to do with the CLI" include the results. And maybe encourage pasting it in a gist rather than polluting the issues?